gitpod-init.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+echo "Installing the GitHub CLI"
+brew install gh
```

Comment:
As mentioned in the PR description, Gitpod supports `brew`.

Reply:
yes, please.